From 607b896b74783149aaf85fe4f32ce8cad75cd245 Mon Sep 17 00:00:00 2001
From: Ajay Panicker <apanicke@google.com>
Date: Thu, 17 Nov 2016 15:50:36 -0800
Subject: [PATCH] Remove MANAGE_DOCUMENTS permission as it isn't needed

Fix Merge Conflict in nyc-mr1-security-c-release
Bug: 32612586
Change-Id: Iebe7dc9b198ce99d5921f5a349ba9d5d4424d156
(cherry picked from commit 22d032fc493070d3c4b2efffa6aa592e20cf8578)
(cherry picked from commit cbced755b72357affca9e785be8384263334833c)
---
 AndroidManifest.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index bb6acf5..ab16f7a 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -62,7 +62,6 @@
     <uses-permission android:name="android.permission.READ_SMS" />
     <uses-permission android:name="android.permission.WRITE_SMS" />
     <uses-permission android:name="android.permission.MEDIA_CONTENT_CONTROL" />
-    <uses-permission android:name="android.permission.MANAGE_DOCUMENTS" />
     <uses-permission android:name="android.permission.UPDATE_APP_OPS_STATS" />
     <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.DEVICE_POWER" />
